docs/source/conf.py

```diff
@@ -98,7 +98,7 @@
 copyright = u'2014-2015, Red Hat'
 
 version = VERSION
-release = 0
+release = '0'
```

Comment:
Yep...
~~~
======================================================================
ERROR: test_build_docs (__main__.DocBuildTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/apahim/src/avocado/selftests/doc/test_doc_build.py", line 77, in test_build_docs
    raise DocBuildError(e_msg)
DocBuildError: 1 ERRORS and/or WARNINGS detected while building the html docs:
1) WARNING: The config value `release' has type `int', defaults to `str.'
~~~